doc/api/module.md

```diff
@@ -285,20 +285,22 @@ const { register } = require('node:module');
 const { pathToFileURL } = require('node:url');
 
 const parentURL = pathToFileURL(__filename);
-register('./first.mjs', parentURL);
-register('./second.mjs', parentURL);
+register('./foo.mjs', parentURL);
+register('./bar.mjs', parentURL);
 import('./my-app.mjs');
 ```
 
-In this example, the registered hooks will form chains. If both `first.mjs` and
-`second.mjs` define a `resolve` hook, both will be called, in the order they
-were registered. The same applies to all the other hooks.
+In this example, the registered hooks will form chains. These chains run
+last-in, first out (LIFO). If both `foo.mjs` and `bar.mjs` define a `resolve`
+hook, they will be called like so (note the right-to-left):
+node's default ← `./foo.mjs` ← `./bar.mjs`. The same applies to all the other
```

Comment:
English is left-to-right, so it should be written that way.
```suggestion
hook, they will be called like so: `./bar.mjs`, then `./foo.mjs`, then the Node.js default.
The same applies to all the other
```

Reply:
I don’t find an ASCII diagram easier to follow that plain English “first this gets called, and its result is passed into that, and that result is passed into this other thing.” You can add a diagram too if you want, but I think the straightforward English is more important.

Also why not have the arrows in the diagram point the other way? Then the English and the diagram match.